server/fleet/activities.go

```diff
@@ -13,6 +13,12 @@ const (
 	ActivityTypeDeletedPack = "deleted_pack"
 	// ActivityTypeAppliedSpecPack is the activity type for pack specs applied
 	ActivityTypeAppliedSpecPack = "applied_spec_pack"
+	// ActivityTypeCreatedPolicy is the activity type for created policies
+	ActivityTypeCreatedPolicy = "created_policy"
+	// ActivityTypeEditedPolicy is the activity type for edited policies
+	ActivityTypeEditedPolicy = "edited_policy"
+	// ActivityTypeDeletedPolicy is the activity type for deleted policies
+	ActivityTypeDeletedPolicy = "deleted_policy"
```

Comment:
We are missing the activity for `ActivityTypeAppliedSpecPolicy` (similar to `ActivityTypeAppliedSpecSavedQuery`).